### PR TITLE
Copy update

### PR DESCRIPTION
**Pull request by rjlynch**

### CODE REVIEW

File: app/views/early_years_payment/practitioner/claims/how_we_use_your_information.html.erb [hunk 1]
```diff
@@ -1,52 +1,123 @@
-<% title = "How we’ll use the information you provide" %>
+<% title = "How we'll process your claim" %>
```
Comment (asmega) [inline app/views/early_years_payment/practitioner/claims/how_we_use_your_information.html.erb:1]: smart quotes + was this not the one you reverted?

Comment (rjlynch) [reply app/views/early_years_payment/practitioner/claims/how_we_use_your_information.html.erb]: Sorted the smart quotes.
Yeah this was the one I reverted as it hadn't been business approved

**Review by asmega — APPROVED** [review]
there are more instance of the smart quotes though, i didn't feel the need indicate every single one though 😅

**Comment by rjlynch** [comment]
> there are more instance of the smart quotes though, i didn't feel the need indicate every single one though 😅

I copied the copy straight from the prototype 😂 I'll fix up those quotes though

**Pull request merged**